http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1234,7 +1279,7 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""persistence.postgres.name=${namespaceName.value}-${config.galaxyDiskConfig.postgresPersistenceName}""",
       raw"""galaxy.postgresql.galaxyDatabasePassword=${config.galaxyAppConfig.postgresPassword}""",
       raw"""persistence.postgres.persistentVolume.extraSpec.gcePersistentDisk.pdName=${getGalaxyPostgresDiskName(
-        namespaceName
+        nfsDisk.name
```

Comment:
we might need to the disk name depending on which style disk actually exists in google if we're worried about users might have an older style disk....but at the same time, Gabriela's UI restore PR is not merged yet, so we probably don't really need to worry about this much